backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/Mailer.kt

```diff
@@ -46,6 +46,14 @@ object Mailer {
         }
     }
 
+    private fun EmailBody.applicationConfirmationNoteParagraph(note: String?) {
+        if (!note.isNullOrEmpty()) {
+            p {
+                +"$note"
```

Comment:
1. any smart idea how to render `\n` within a string as line break without splitting the string and rendering a `br()` instead of `\n`